packages/babel-plugin-transform-es2015-block-scoping/README.md

```diff
@@ -2,6 +2,32 @@
 
 > Compile ES2015 block scoping (const and let) to ES5
 
+## Examples
+
+**In**
+
+```javascript
+for (let i=1; i<=10; i++) {
+    setTimeout( function logger(){
+        console.log( i ); // block scope for closure
+    }, i*1000 );
+}
+```
+
+**Out**
+
+```javascript
+var _loop = function _loop(i) {
+    setTimeout(function logger() {
+        console.log(i);
+    }, i * 1000);
+};
+
+for (var i = 1; i <= 10; i++) {
+    _loop(i);
+}
+```
```

Comment:
The current one is good but to complex IMO. My example above shows the basic transformation.